decidim-core/app/cells/decidim/tos_page/refuse_btn_modal.erb

```diff
@@ -1,7 +1,8 @@
-<button class="clear button secondary button--nomargin small" type="button" data-open="tos-refuse-modal">
+<button class="button sm transparent secondary" type="button" data-open="tos-refuse-modal">
   <%= t("refuse.modal_button", scope: "decidim.pages.terms_and_conditions") %>
 </button>
 
+<%# REDESIGN_PENDING: missing design modal %>
```

Comment:
I think it is because of the missing modal design but the modal content is "flashing" within the acceptance section when the page is loaded and I guess it's moved to the bottom of the page by Foundation JS at the beginning.

I understand this may be because of the missing modal design but just wanted to point it out that in the final version we should not have it flashing on page load.

Reply:
All modal-related content will be faced later. Meanwhile we track those down through the REDESIGN_PENDING comments